src/utils.c

```diff
@@ -1693,6 +1695,12 @@ void vctrs_init_utils(SEXP ns) {
   strings_check_unique = Rf_mkChar("check_unique");
   SET_STRING_ELT(strings2, 10, strings_check_unique);
 
+  strings_unique_quiet = Rf_mkChar("unique_quiet");
+  SET_STRING_ELT(strings2, 23, strings_unique_quiet);
+
+  strings_universal_quiet = Rf_mkChar("universal_quiet");
+  SET_STRING_ELT(strings2, 24, strings_universal_quiet);
+
```

Comment:
Nope you did it exactly as I would have done it, just appending to avoid making mistakes.

Worth noting that in the future all of this will be moved to `globals.c`, see https://github.com/r-lib/vctrs/blob/a0ee85860d94459bf83194e036138f3160329cdc/src/globals.c#L64-L66